### PR TITLE
Rework styling/components to support multiple charts on the dashboard

### PR DESCRIPTION
**Pull request by tylershelton**
### CSS
- There are now multiple sass files, so I re-organized them into their own subfolder (`assets/styles`).
- Styling has been iterated on to help support having an arbitrary number of charts on the dashboard. Namely, flexbox is now the main tool doing the layout lift.

### JSX
- There were some changes to component naming and hierarchy --
  - Wrapped `App` in a [StrictMode](https://react.dev/reference/react/StrictMode) container to get better error reporting when in dev mode.
  - `ChartContainer` has been removed entirely; as it wound up fulfilling the same purpose as `Main`.
  - `Main` has been renamed to `Dashboard` for clarity, now that there are other panel views being developed.

### Dev Tooling
- The devspace dev container for kubernautics has been assigned increased resources (CPU and RAM) to improve setup times, webpack build times, and avoid instabilities/crashes that were being caused by a lack of access to memory.

## Up Next
Will continue working to make the multiple charts distinct from one another -- right now they are all hard-coded duplicates of one another. Wanted to get this PR in without waiting for that so everyone had the updated react component hierarchy to work with.

### CODE REVIEW

File: devspace.yaml
Comment (edli-dot) [inline devspace.yaml:1]: This was a brilliant find. Initial load times for devspace kubernautics is a lot faster now as well as when in web pack dev mode.

Comment (edli-dot) [reply devspace.yaml:1]: Good stuff Tyler. Components have been compiled. Charts on dashboard adjust according to the size of the browser window. I'm taking it that going forward, we might create additional sass files and just pass it into 'styles.scss' to help modularize the styling? I'm totally for it.

**Review by edli-dot — APPROVED** [review]
Good stuff Tyler. Redundant components have been consolidated. Charts on dashboard adjust according to the size of the browser window. I'm taking it that going forward, we might create additional sass files and just pass it into 'styles.scss' to help modularize the styling? I'm totally for it.

File: devspace.yaml
Comment (edli-dot) [reply devspace.yaml:1]: *When you get the chance, I would love to hear what made you think to wrap the application in StrictMode*

**Comment by tylershelton** [comment]
Thanks! Yeah, I think having separate sass files for different areas of the app will make the styling more maintainable.

**Pull request merged**